lib/magma/attribute.rb

```diff
@@ -86,10 +101,6 @@ def display_name
       @display_name ||= name.to_s.split(/_/).map(&:capitalize).join(' ')
     end
 
-    def description
-      @description || @desc
-    end
```

Comment:
`@desc` was hanging around as a fallback but we don't need anymore because IPI is already using `description`.